core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -1121,6 +1121,10 @@ public ChangeGraphResponse changeGraph(Collection<JsonFeature> collection) {
         }
     }
 
+    public String getBlockedRectangularAreas() {
+        return blockedRectangularAreas;
```

Comment:
Do we really need to expose this :) ? I would like to hide it as long as possible as the API for the blocked areas is not really stable and e.g. the String could change to a domain object etc to be more accessible. Also as this is not related to this PR we should move this to another issue?